Source/WTF/Scripts/Preferences/WebPreferencesInternal.yaml

```diff
@@ -289,7 +289,8 @@ EventHandlerDrivenSmoothKeyboardScrollingEnabled:
     WebKitLegacy:
       default: false
     WebKit:
-      "PLATFORM(COCOA)": true
+      "PLATFORM(MAC)": WebKit::defaultScrollAnimatorEnabled()
+      "PLATFORM(IOS)": true
```

Comment:
This should be `PLATFORM(IOS_FAMILY)` since `PLATFORM(COCOA)` is `PLATFORM(MAC) || PLATFORM(IOS_FAMILY)`.

@hortont424 is it expected that `EventHandlerDrivenSmoothKeyboardScrollingEnabled` was true on iOS?

Reply:
Seems like a surprising choice :) I don't recall how WKKeyboardScrollingAnimator wins....

Reply:
should I change it to be false by default and only then `WebKit::defaultScrollAnimatorEnabled()` for MAC?